pkg/apis/controlplane/types.go

```diff
@@ -261,9 +264,9 @@ type NodeStatsSummary struct {
 	// The TrafficStats of K8s NetworkPolicies collected from the Node.
 	NetworkPolicies []NetworkPolicyStats
 	// The TrafficStats of Antrea ClusterNetworkPolicies collected from the Node.
-	AntreaClusterNetworkPolicies []NetworkPolicyStats
+	AntreaClusterNetworkPolicies []AntreaNetworkPolicyStats
```

Comment:
This is incompatible change. We should still use NetworkPolicyStats and only add RuleTrafficStats to it.

Reply:
Is NodeStatsSummary an internal API? Do we need to keep the internal API unchanged? From the current implementation, Antrea agents only collect the rule level policies and won't addUp to the policy level.

Reply:
It's not about internal or external API, it is to keep backwards-compatible between components. Imagine that you upgrade antrea-controller, which expects `AntreaClusterNetworkPolicies` to be a slice of `AntreaNetworkPolicyStats`, but antrea-agent will be rolling updated, so some instances may still use the old version API for long time, which sets `AntreaClusterNetworkPolicies` as a slice of `NetworkPolicyStats`. Besides, Antrea commits version skew between antrea-controller and antrea-agent: https://github.com/vmware-tanzu/antrea/blob/main/docs/versioning.md#antrea-upgrade-and-supported-version-skew so it may not only when upgrading but also be long running.

Reply:
Thanks for the detailed explanation, I have changed back to NetworkPolicyStats and add RuleTrafficStats to it.